replay_build_scripts/common.mjs

```diff
@@ -43,6 +43,20 @@ export function currentPlatform() {
   }
 }
 
+export function outputArchitecture() {
```

Comment:
This is not the ideal way for us to handle architecture but it's how we effectively do it today.